### PR TITLE
+angstrom.0.14.0

### PR DESCRIPTION
**Pull request by seliopou**

### CODE REVIEW

**Comment by kit-ty-kate** [comment]
The new interface brought in this new release apparently broke lots of packages. I'll do a PR to add the proper constraints to those and I'll relaunch the CI

**Comment by seliopou** [comment]
@dinosaure this probably affects a few of your libraries. You can preserve existing behavior by turning `parse_string` to `parse_string ~consume:Prefix`. However, you should consider changing that to `parse_string ~consume:All` instead. Same goes for `parse_bigstring`.

File: packages/angstrom-unix/angstrom-unix.0.14.0/opam [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+dev-repo: "git+https://github.com/inhabitedtype/angstrom.git"
+build: [
+  ["dune" "subst"] {pinned}
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
+]
+depends: [
+  "ocaml" {>= "4.03.0"}
+  "dune" {>= "1.0"}
+  "angstrom"
```
Comment (kit-ty-kate) [inline packages/angstrom-unix/angstrom-unix.0.14.0/opam:16]: Are these dependencies not supposed to be synchronized with the version of angstrom? I would highly suggest to add `{= version}` here

File: packages/angstrom-lwt-unix/angstrom-lwt-unix.0.14.0/opam [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+dev-repo: "git+https://github.com/inhabitedtype/angstrom.git"
+build: [
+  ["dune" "subst"] {pinned}
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
+]
+depends: [
+  "ocaml" {>= "4.03.0"}
+  "dune" {>= "1.0"}
+  "angstrom"
```
Comment (kit-ty-kate) [inline packages/angstrom-lwt-unix/angstrom-lwt-unix.0.14.0/opam:16]: Are these dependencies not supposed to be synchronized with the version of angstrom? I would highly suggest to add `{= version}` here

File: packages/angstrom-async/angstrom-async.0.14.0/opam [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+dev-repo: "git+https://github.com/inhabitedtype/angstrom.git"
+build: [
+  ["dune" "subst"] {pinned}
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
+]
+depends: [
+  "ocaml" {>= "4.04.1"}
+  "dune" {>= "1.0"}
+  "angstrom" {>= "0.7.0"}
```
Comment (kit-ty-kate) [inline packages/angstrom-async/angstrom-async.0.14.0/opam:16]: Are these dependencies not supposed to be synchronized with the version of angstrom? I would highly suggest to add `{= version}` here

**Comment by kit-ty-kate** [comment]
I've fixed the remaining revdeps in a separate PR, sorry for the time it took me to do it.
Once this is merged and my latest comments taken care of, it should be good to go.

**Comment by kit-ty-kate** [comment]
All green now. This can be merged once the questions have been taken care of

File: packages/angstrom-unix/angstrom-unix.0.14.0/opam (hunk 1 above)
Comment (seliopou) [reply packages/angstrom-unix/angstrom-unix.0.14.0/opam:16]: This is intentional. I only add constraints when API changes require them.

File: packages/angstrom-lwt-unix/angstrom-lwt-unix.0.14.0/opam (hunk 2 above)
Comment (seliopou) [reply packages/angstrom-lwt-unix/angstrom-lwt-unix.0.14.0/opam:16]: This is intentional. I only add constraints when API changes require them.

File: packages/angstrom-async/angstrom-async.0.14.0/opam (hunk 3 above)
Comment (seliopou) [reply packages/angstrom-async/angstrom-async.0.14.0/opam:16]: This is intentional. I only add constraints when API changes require them.

**Comment by seliopou** [comment]
Should be good! Thanks for finishing up the constraints. I didn't occur to me that so many packages would break.

**Comment by kit-ty-kate** [comment]
Thanks!

**Pull request merged**